src/main/java/mate/academy/hibernate/relations/model/Country.java

```diff
@@ -11,6 +19,8 @@ public Country(String name) {
         this.name = name;
     }
 
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
```

Comment:
it's not a field. It's a method. But you should add this annotation for the fields